modules/core/eks/modulestack.yaml

```diff
@@ -23,6 +23,17 @@ Resources:
               - "sts:AssumeRole"
             Resource:
               - !Sub 'arn:aws:iam::${AWS::AccountId}:role/addf-${DeploymentName}-${ModuleName}-${AWS::Region}-masterrole'
+          - Effect: Allow
+            Action:
+              - "ssm:GetParameter"
+            Resource:
+              - "*"
+          - Effect: Allow
+            Action:
+              - "s3:GetObject"
+              - "s3:ListBucket"
+            Resource:
+              - "*"
```

Comment:
this seems too wide open.  Do you really need GetObject over ALL S3 buckets?

Reply:
Done